app/models/applicationPackage.ts

```diff
@@ -0,0 +1,32 @@
+import { Record } from "immutable";
```

Comment:
I will skip this one and do all the rest of the models in one go as a separate PR. Have done the forms.